build.py

```
@@ -55,25 +55,335 @@ def _load_releases() -> dict[str, str]:
+                    before_pipe = line.split("|")[0].strip()
+                    parts = before_pipe.split()
+                    if parts:
+                        addr_candidate = parts[0].strip()
+                        if addr_candidate and addr_candidate != "00000000":
+                            addr = addr_candidate
+                            break
+
+    if addr:
+        # Feed address + binary to llvm-symbolizer via stdin
+        input_str = f"0x{addr}\n{test_bin}\n"
+        result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)